spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc

```diff
@@ -69,7 +69,7 @@ deployment to fail
 
 [subs=attributes]
 ```
-cf push dataflow-server -m 2G -k 2G --no-start -p spring-cloud-dataflow-server-cloudfoundry-{project-version}.jar
+cf push dataflow-server -b java_buildpack_offline -m 2G -k 2G --no-start -p spring-cloud-dataflow-server-cloudfoundry-{project-version}.jar
```

Comment:
`cf buildpacks` doesn't list `java_buildpack_offline` as a known buildpack. May be it is just for the CF environment I target to. Also, not sure which buildpacks get added by default for any CF environment. May be we can use the one (`java_buildpack` ?) that comes from default?